datadog/resource_datadog_dashboard.go

```diff
@@ -3000,6 +3015,12 @@ func buildDatadogTimeseriesDefinition(terraformDefinition map[string]interface{}
 	if v, ok := terraformDefinition["marker"].([]interface{}); ok && len(v) > 0 {
 		datadogDefinition.Markers = *buildDatadogWidgetMarkers(&v)
 	}
+	if v, ok := terraformDefinition["event"].([]interface{}); ok && len(v) > 0 {
+		datadogDefinition.Events = *buildDatadogWidgetEvents(&v)
+	}
+	if v, ok := terraformDefinition["yaxis"].(map[string]interface{}); ok && len(v) > 0 {
```

Comment:
Since `yaxis` is no longer a `TypeMap`, `terraformDefinition["yaxis"]` should be cast to `[]interface{}` instead.